solutions/bio1/BlockChains/index.tsx

```diff
@@ -0,0 +1,58 @@
+﻿import { SCodeBlock, SText, STitle, SList } from "components";
+//import { brace, floor } from "solutions/util";
+
+export const BlockChains = () => {
+    return (
+        <>
+            <STitle>Observation</STitle>
+            <SText>
+                A straightforward way to solve this problem is to generate all possible permutations of the blocks and then filter out those that violate the no-three-in-order condition.
+                Specifically, we would:
+            </SText>
+            <SList>
+                <SText> 1. Generate all possible permutations of the remaining letters that could follow the prefix p.</SText>
```

Comment:
This renders both a bullet point and a number.

Instead, open with `<SList numbered>` and don't write `1.` or `2.` or `3.` on each list item.

Reply:
Also, I would write p in LaTeX here (`$p$`) since it's a variable given in the input.